autogpt_platform/frontend/src/components/agptui/FeaturedAgentCard.tsx

```diff
@@ -27,16 +27,20 @@ export const FeaturedAgentCard: React.FC<FeaturedStoreCardProps> = ({
       data-testid="featured-store-card"
       onMouseEnter={() => setIsHovered(true)}
       onMouseLeave={() => setIsHovered(false)}
-      className={backgroundColor}
+      className={`flex h-[482px] w-[440px] flex-col ${backgroundColor}`}
     >
-      <CardHeader>
-        <CardTitle>{agent.agent_name}</CardTitle>
-        <CardDescription>{agent.description}</CardDescription>
```

Comment:
The agent object should have a sub heading on it, which is what needs to be used. Though, this may have been lost when changing from FeaturedAgent to StoreAgent object.